src/components/filters/FilterSection/FilterSection.css

```diff
@@ -189,3 +215,56 @@
   justify-content: center;
   align-items: center;
 }
+
+.g3-icon--question-mark-bootstrap.help-tooltip-icon {
+  background-color: black;
+  top:-3px;
+  opacity:0.9;
+  width: 14px;
+  height: 14px;
+  position:relative;
+}
+
+.g3-tooltip {
+  margin-left:10px;
+  position:relative;
+  display:inline-block;
+}
+
+.g3-tooltip-body {
```

Comment:
we already have some predefined styles for tooltip on buttons https://github.com/uc-cdis/gen3-ui-component/blob/master/src/components/Button/Button.css#L117 which has a light bg color

we should stick to this so we don't have a bunch of tooltips in different colors